apps/web/app/(app)/(survey-editor)/environments/[environmentId]/surveys/[surveyId]/edit/components/SelectQuestionForm.tsx

```diff
@@ -318,7 +316,6 @@ export const MultipleChoiceSingleForm = ({
               variant="minimal"
               type="button"
               onClick={() => {
-                // @ts-expect-error
                 updateQuestion(questionIdx, { type: TSurveyQuestionType.MultipleChoiceMulti });
               }}>
               Convert to Multi Select
```

Comment:
Please modify this button to change the type of the question(single to multi and vice versa).